js/components/conceptsetbuilder/components/IncludedConcepts.js

```diff
@@ -159,24 +159,24 @@ define([
 					break;
 			}
 		}
-		
+
 		// behaviors
-		
+
 		self.refresh = function() {
 			self.isLoading(true);
 			self.selectedConcepts([]);
-			VocabularyAPI.resolveConceptSetExpression(self.conceptSet()).then(function (identifiers) {
+			VocabularyAPI.resolveConceptSetExpression(self.conceptSet()).done(function (identifiers) {
```

Comment:
My understanding on `promise.done` is that it's not an official syntax.  @pavgra , can you confirm that? I thought we saw an issue with the bundler that done was stripped out, but we polly-filled it in. Should we depend on the polyfill or avoid using `.done()`?

Reply:
Actually resolveConceptSetExpression using jquery.ajax under the hood. We talk about refactoring with @pavgra and decided not to make any refactoring in this rc branch.